src/utilities/sequence-editor/sequence-linter.ts

```diff
@@ -179,118 +173,54 @@ export function sequenceLinter(
     return diagnostics;
   }
 
-  function conditionalAndLoopKeywordsLinter(commandNodes: SyntaxNode[], text: string): Diagnostic[] {
```

Comment:
Does this new code deprecate the existing conditional and loop keywords that exist in the adaptation, or is there just a follow up ticket to move the new keywords you defined into the adaptation and change the structure?

Reply:
It's redundant with some existing adaptation code, and some of this code may move to adaptation, but I'd like to work out the VML support first to see what code can stay in core.

This doesn't break any adaptation code afaik